### ISSUE TITLE
Laravel custom larnguage file

### ISSUE THREAD

**Issue by hamidafghan** [hekmatinasser/verta#165]
Based on https://github.com/hekmatinasser/jalali/issues/9 issue, I tried to set the message in the provider but it looks like the function `setMessages` isn't static. 

So basically the target is to register the custom messages in `AppServiceProvider` so all object's messages should be based on registered local messages. 

```php

class AppServiceProvider extends ServiceProvider
{
    public function boot(): void
    {
        // Set messages of Verta
       Verta::setMessages('fa', ['months.12' => 'حوت']);
    }
}
```

But it throws the error, `  Non-static method Hekmatinasser\Jalali\Jalali::setMessages() cannot be called statically`, would you provide an example how to do it in provider (globally).

Thanks

**Comment by hamidafghan** [comment]
https://github.com/hekmatinasser/jalali/pull/10

**Comment by hamidafghan** [comment]
Any support here? Thanks

**Comment by hamidafghan** [comment]
Dear @hekmatinasser, any chance to have a look at this issue? Thanks

**Comment by hekmatinasser** [comment]
fix jalali 8.2.3 
thank you @hamidafghan
